cypress/e2e/WebInterface/Test Cases/QI-CORE Test Case/Execution/TestCaseExecutionWithCode.cy.ts

```diff
@@ -45,6 +44,7 @@ describe('Test Case Execution with codes', () => {
 
         cy.get(EditMeasurePage.cqlEditorSaveButton).click()
         cy.get(CQLEditorPage.successfulCQLSaveNoErrors).should('be.visible')
+        cy.wait(2000)
```

Comment:
any chance we can do this without an explicit wait? Can we wait for the next element to be visible and enabled or something like that?